ci-operator/step-registry/cucushift/upgrade/disconnected/mirrorimage/cucushift-upgrade-disconnected-mirrorimage-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Can we reuse `ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh`?

Reply:
Good question. What I'm concerned are
1. We will support one more hops upgrade. Then we need to iterate over all the target images in the step.
2. The mirror-images-by-oc-adm-commands.sh mirrors RELEASE_IMAGE_LATEST which is the source image rather than target image in the upgrade test.
So I'm introducing a separate step for upgrade.

Reply:
For 1, will this ref will be call for multiple times, or in this script, will introduce a loop for that?

Reply:
I'd prefer a loop in the script.

Reply:
okay. got it.